local-tests/test.ts

```diff
@@ -104,6 +104,12 @@ import { testFailImportWrappedKeysWithExpiredSessionSig } from './tests/wrapped-
 import { testExportWrappedKey } from './tests/wrapped-keys/testExportWrappedKey';
 import { testSignMessageWithSolanaEncryptedKey } from './tests/wrapped-keys/testSignMessageWithSolanaEncryptedKey';
 import { testSignTransactionWithSolanaEncryptedKey } from './tests/wrapped-keys/testSignTransactionWithSolanaEncryptedKey';
+import { testBatchGeneratePrivateKeys } from './tests/wrapped-keys/testBatchGeneratePrivateKeys';
+
+import { setLitActionsCodeToLocal } from './tests/wrapped-keys/util';
+
+// Use the current LIT action code to test against
+setLitActionsCodeToLocal();
```

Comment:
This makes our tinny tests use the locally built code for all LIT actions in wrapped-keys instead of running them against the CIDs in `wrapped-keys` LIT_ACTION_CID_REPOSITORY

Reply:
Since the idea is that users use the IPFS id shouldn't the tests reflect that?

Reply:
I guess it depends on the users; some people will be using the code directly, others the CIDs... what I want to do is run the entire suite against the cids and then again against the current code ideally, but see other comment re: doing so without jest.

The reason this is here right now is because we have a chicken and egg problem - in-progress code in PRs isn't in IPFS, and pinning repeatedly to IPFS as we iterate on a lit action would result in a ton of churn and manual updates right now.

We could make this code always use IPFS CIDs, but it would require that all actions we want to test would exist in IPFS before we can test against them -- and we want to run tests while developing the actions where they will be changing often.  

In this PR, I wanted to be sure tests pass with the _current code_ rather than previous CIDs -- once we have confirmed all of the LIT actions code here is GTG and published all of the lit actions to IPFS, I can update the CIDs in `wrapped-keys` and remove this code so the tests run against CIDs again

Reply:
:+1: I agree @FedericoAmura <3 Ideally, all tests would run against the data in the same commit by _default_, and if we want to test against prior versions we'd explicitly do _that_.  The main thing is just getting the CID repo in the same commit to always match the current source code in that commit... I think we can do that with a husky hook script that runs on-commit

Reply:
> husky hook script that runs on-commit

Good idea